packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -845,10 +851,174 @@ export default class Meeting extends StatelessWebexPlugin {
      */
     this.transcription = undefined;
 
+    /**
+     * Password status. If it's PASSWORD_STATUS.REQUIRED then verifyPassword() needs to be called
+     * with the correct password before calling join()
+     * @instance
+     * @type {PASSWORD_STATUS}
+     * @public
+     * @memberof Meeting
+     */
+    this.passwordStatus = PASSWORD_STATUS.UNKNOWN;
+
+    /**
+     * Information about required captcha. If null, then no captcha is required. status. If it's PASSWORD_STATUS.REQUIRED then verifyPassword() needs to be called
+     * with the correct password before calling join()
+     * @instance
+     * @type {Object}
+     * @property {string} captchaId captcha id
+     * @property {string} verificationImageURL Url of the captcha image
+     * @property {string} verificationAudioURL Url of the captcha audio file
+     * @property {string} refreshURL Url used for refreshing the captcha (don't use it directly, call refreshCaptcha() instead)
+     * @public
+     * @memberof Meeting
+     */
+    this.requiredCaptcha = null;
+
+    /**
+     * Indicates the reason for last failure to obtain meeting.meetingInfo. MEETING_INFO_FAILURE_REASON.NONE if meeting info was
+     * retrieved successfully
+     * @instance
+     * @type {MEETING_INFO_FAILURE_REASON}
+     * @private
+     * @memberof Meeting
+     */
+    this.meetingInfoFailureReason = undefined;
+
     this.setUpLocusInfoListeners();
     this.locusInfo.init(attrs.locus ? attrs.locus : {});
   }
 
+  /**
+   * Fetches meeting information.
+   * @param {Object} options
+   * @param {String} options.destination
+   * @param {String} options.type
+   * @private
+   * @memberof Meeting
+   * @returns {Promise}
+   */
+  async fetchMeetingInfo({
+    destination, type, password = null, captchaCode = null
+  }) {
+    if (captchaCode && !this.requiredCaptcha) {
+      return Promise.reject(new Error('captcha not required'));
+    }
+    if (password && (this.passwordStatus !== PASSWORD_STATUS.REQUIRED && this.passwordStatus !== PASSWORD_STATUS.UNKNOWN)) {
+      return Promise.reject(new Error('password not required'));
```

Comment:
Can you make this error more specific? when looking at the logs this is not helpful

Reply:
done